src/components/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import { ReactNode } from "react";
+
```

Comment:
You have to develop pixel perfect what is on figma design.
It means that this component should be responsive.